docs/examples/timer_operations.rst

```diff
@@ -51,7 +74,14 @@ define our recurring Timer job.
     start = datetime.utcnow()
     interval = timedelta(minutes=30)
     name = "sdk-timer-example"
-    job = TimerJob.from_transfer_data(data, start, interval, stop_after_n=2, name=name)
+    job = TimerJob.from_transfer_data(
+        data,
+        start,
+        interval,
+        stop_after_n=2,
+        name=name,
+        scope=str(transfer_action_provider_scope),
```

Comment:
To clarify, this is passing the string-ized full dependency list. 
Is it also valid to pass Timer only this?
`scope="https://auth.globus.org/scopes/actions.globus.org/transfer/transfer"`

It seems to me that if Timer is going to take its token, do a dependent token grant, and then lookup values in the response, we really ought to be passing it the resource server name ("actions.globus.org", I think?) rather than a scope. Or is there some other purpose here, which requires the full scope be passed?

Reply:
Yeah, good call—I had that wrong!